### ISSUE TITLE
Pre-compiled binaries for Linux / R Wrapper

### ISSUE THREAD

**Issue by BirgerNi** [CoolProp/CoolProp#2021]
### Description

There are no pre-compiled binaries for `R` in the current release:

![grafik](https://user-images.githubusercontent.com/21315429/117273154-240d2e80-ae5c-11eb-847d-650c4307fd2b.png)

However, there were binaries in earlier versions:

![grafik](https://user-images.githubusercontent.com/21315429/117273296-4acb6500-ae5c-11eb-8f85-7039d168add2.png)

Are Linux user expected to build from source?

### Steps to Reproduce

1. Current release: https://sourceforge.net/projects/coolprop/files/CoolProp/6.4.1/R/
2. Link to an older release: https://sourceforge.net/projects/coolprop/files/CoolProp/5.1.1/R/

**Comment by ibell** [comment]
Yes, we dropped the linux binaries.  It is very hard to keep up with the binaries on linux given the variety of different options. Had to prioritize the most important binaries.  If you wanted to dedicate a machine to building binaries and commit to keeping it operational, we could consider to add it back.  Generally linux users are savvy enough to build things on their own; our instructions are pretty clear I think, and fixes to the building docs are welcome.

**Comment by BirgerNi** [comment]
That's ok. I guess I'll install the python package and access that from `R` via the `R` package `reticulate`. You may close this issue.

**Comment by ibell** [comment]
Please go ahead and close the issue if you are satisfied.

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
